packages/store/addon/-private/ts-interfaces/record-data-record-wrapper.ts

```diff
@@ -39,7 +38,7 @@ export default interface RecordDataRecordWrapper {
   /**
    * @deprecated
    */
-  getErrors?({}): JsonApiValidationError[];
+  getErrors?({}): JsonApiValidationError[]; // eslint-disable-line no-empty-pattern
```

Comment:
I think this is actually a case where we're seeing false positives because the rule was originally not implemented with TypeScript in mind, so eslint is interpreting the type definition as an attempt to destructure.

I will open an issue with @typescript-eslint